azure-iot-device/azure/iot/device/common/pipeline/pipeline_ops_mqtt.py

```diff
@@ -45,6 +46,7 @@ def __init__(
         self.ca_cert = ca_cert
         self.client_cert = client_cert
         self.sas_token = sas_token
+        self.websockets = websockets
```

Comment:
>self.websockets = websockets [](start = 8, length = 28)

Is this getting used anywhere ? I think you are directly passing this into MQTTTransport from the pipeline root

Reply:
good catch